src/main/java/frc/trigon/robot/subsystems/climber/placeholderclimber/PlaceholderClimberIO.java

```diff
@@ -0,0 +1,57 @@
+package frc.trigon.robot.subsystems.climber.placeholderclimber;
+
+import com.ctre.phoenix6.BaseStatusSignal;
+import com.ctre.phoenix6.controls.MotionMagicVoltage;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.signals.NeutralModeValue;
+import frc.trigon.robot.subsystems.climber.ClimberConstants;
+import frc.trigon.robot.subsystems.climber.ClimberIO;
+import frc.trigon.robot.subsystems.climber.ClimberInputsAutoLogged;
+import frc.trigon.robot.utilities.Conversions;
+
+public class PlaceholderClimberIO extends ClimberIO {
+    private final TalonFX
+            masterMotor = PlaceholderClimberConstants.MASTER_MOTOR,
+            followerMotor = PlaceholderClimberConstants.FOLLOWER_MOTOR;
+    private final MotionMagicVoltage
+            masterMotorPositionRequest = new MotionMagicVoltage(0).withEnableFOC(PlaceholderClimberConstants.ENABLE_FOC),
+            followerMotorPositionRequest = new MotionMagicVoltage(0).withEnableFOC(PlaceholderClimberConstants.ENABLE_FOC);
+
+    @Override
+    protected void updateInputs(ClimberInputsAutoLogged inputs) {
+        refreshStatusSignals();
+        inputs.encoderPositionMeters = Conversions.revolutionsToDistance(PlaceholderClimberConstants.ENCODER_POSITION_SIGNAL.getValue(), ClimberConstants.DIAMETER_METERS);
+        inputs.encoderVelocityMetersPerSecond = Conversions.revolutionsToDistance(PlaceholderClimberConstants.ENCODER_VELOCITY_SIGNAL.getValue(), ClimberConstants.DIAMETER_METERS);
+        inputs.motorProfiledSetpointMeters = Conversions.revolutionsToDistance(PlaceholderClimberConstants.MOTOR_SETPOINT_SIGNAL.getValue(), ClimberConstants.DIAMETER_METERS);
+        inputs.motorVoltage = PlaceholderClimberConstants.MOTOR_VOLTAGE_SIGNAL.getValue();
+        inputs.motorCurrent = PlaceholderClimberConstants.MOTOR_CURRENT_SIGNAL.getValue();
+    }
+
+    @Override
+    protected void setPositionMeters(double targetPositionMeters) {
+        masterMotor.setControl(masterMotorPositionRequest.withPosition(Conversions.distanceToRevolutions(targetPositionMeters, ClimberConstants.DIAMETER_METERS)));
+        followerMotor.setControl(followerMotorPositionRequest.withPosition(Conversions.distanceToRevolutions(targetPositionMeters, ClimberConstants.DIAMETER_METERS)));
```

Comment:
since you have a master and a follower, you don't need to send requests to the follower